include/powsybl/iidm/converter/ExportOptions.hpp

```diff
@@ -21,58 +22,197 @@ namespace converter {
+     * Add an extension to the list of exported extensions.
+     *
+     * @param extension The name of the extension to add
+     *
+     * @return this ExportOptions object
+     */
     ExportOptions& addExtension(const std::string& extension);
 
+    /**
+     * Add a given version in which the extension with the given name will be exported if this version is supported by
+     * the extension's XML serializer and if it is compatible with the IIDM version in which the network will be exported.
+     *
+     * If the version is not added for an extension configured to be serialized, the extension will be serialized in the
+     * most recent version compatible with the IIDM version in which the network will be exported.
+     *
+     * If a version is added for an extension configured <b>not</b> to be serialized, the version will be ignored.
+     *
+     * If a version has already been added for the extension, throw an exception.
+     *
+     * @param extensionName The name of the extension
+     * @param extensionVersion The version in which the extension should be exported
+     *
+     * @return this ExportOptions object
+     */
+    ExportOptions& addExtensionVersion(const std::string& extensionName, const std::string& extensionVersion);
+
+    /**
+     * Return the expected version for the given extension
+     *
+     * @param extensionName The name of the extension
+     *
+     * @return the version in which the extension should be exported or an empty string if the extension is not found
+     */
+    const std::string& getExtensionVersion(const std::string& extensionName) const;
+
+    /**
+     * Return the XIIDM version to use
+     *
+     * @return the XIIDM version
+     */
     const std::string& getVersion() const;
 
+    /**
+     * Return the maximal topology level to use
+     *
+     * The TopologyLevel is used to decrease the level of detail of a voltage level topology.
+     *
+     * @return the maximal TopologyLevel to use
+     */
     const TopologyLevel& getTopologyLevel() const;
 
+    /**
+     * Return true if the identifier should be anonymized
+     *
+     * @return true if the identifier should be anonymized, false otherwise
+     */
     bool isAnonymized() const;
 
+    /**
+     * Return true if the XML should be indented
+     *
+     * @return true if the XML should be indented, false otherwise
+     */
     bool isIndent() const;
 
+    /**
+     * Return true if the main connected component only should be exported. If false, all the connected components are exported.
+     *
+     * @return true if the main connected component only should be exported, false otherwise.
+     */
     bool isOnlyMainCc() const;
 
+    /**
+     * Return true if an exception should be thrown if an XML serializer for an extension is not found.
+     *
+     * @return true if an exception should be thrown, false otherwise
+     */
     bool isThrowExceptionIfExtensionNotFound() const;
 
+    /**
+     *
+     * @return
```

Comment:
Documentation is missing :)

Reply:
Because I don't know what it does... It seems not use in Java... it's a very old property. @MioRtia do you know what is this parameter?